### DISCUSSION TITLE
Need to access Github Secrets in Fastlane(Fastfile/env) files

### DISCUSSION THREAD

**Discussion by nnvdp** [fastlane/fastlane#21030] [Q&A]
Hello Team,

We would like to access our mobile app git repository github encrypted secrets in Fastfile or env files of Fastlane while executing the lane to upload_to_playstore.

I have gone through the blog at https://www.runway.team/blog/ci-cd-pipeline-android-app-fastlane-github-actions for the reference. But not getting the exact way to get access encrypted secrets in fastlane.

Please note, we don't want to use Github Actions workflow in this case.

Please suggest if there is anyway to achieve the above one.

Thanks & Regards,
Durga Prasad Nallamothu.

**Comment by iBotPeaches** [comment]
You don't want to use GitHub Actions, but you use GitHub, but you don't tell us what you are using. How is anyone supposed to help? If you use GitHub secrets - its pretty much intended that you access them via the GitHub Action workflows. You won't be able to API out secrets from GitHub - you have to use the Actions - https://docs.github.com/en/actions/security-guides/encrypted-secrets
